deepforest/_explainer.py

```diff
@@ -0,0 +1,65 @@
+"""
+Interpretability of the Deep Forest using SHAP:
+    https://github.com/slundberg/shap
+
+A helpful tutorial:
+    https://shap.readthedocs.io/en/latest/example_notebooks/tabular_examples/tree_based_models/Example%20of%20loading%20a%20custom%20tree%20model%20into%20SHAP.html#
+
+"""
+
+import numpy as np
+import scipy
+import shap
+import sklearn
+import graphviz
+
+def get_shap_explainer_forest(forest):
+    """
+    Get the tree explainer for a forest estimator in the deep forest.
+
+    Parameters
+    ----------
+    forest : :obj:`forest`
+        The forest estimator that we want to explain.
+
+    Returns
+    -------
+    TreeExplainer : :obj:`shap.TreeExplainer`
+        Tree explainer for the forest estimator.
+    """
+
+    # Pull the info of the first tree
+    tree_tmp = forest.estimators_[0][0].tree_
```

Comment:
I think the problem is here. The forest structure in DF21 does not use a list `estimators_` to store all decision trees.